### PR TITLE
add git's submodule cheetsheet

### PR DESCRIPTION
**Pull request by vungoccuong**

### CODE REVIEW

**Comment by denisidoro** [comment]
There's a test failing because variables can't contain `-`. Could you replace `-` by `_`, please?

**Comment by vungoccuong** [comment]
yep . i will update now

**Comment by vungoccuong** [comment]
> There's a test failing because variables can't contain `-`. Could you replace `-` by `_`, please?

i updated . i can  check again !!

**Comment by vungoccuong** [comment]
i also updated some docker and docker-compose 's cheatsheet

File: cheats/docker-compose.cheat [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# Builds, (re)creates, starts, and attaches to containers for all services
```
Comment (denisidoro) [inline cheats/docker-compose.cheat]: missing tag: `% docker, docker-compose`

File: cheats/npm.cheat [hunk 2]
```diff
@@ -0,0 +1,20 @@
+# initial new package
```
Comment (denisidoro) [inline cheats/npm.cheat]: missing tag: `% npm, javascript, js`, for example

File: cheats/nvm.cheat [hunk 3]
```diff
@@ -0,0 +1,11 @@
+# install a specified version of node
```
Comment (denisidoro) [inline cheats/nvm.cheat]: missing tag: `% nvm, javascript, js`, for example

File: cheats/git.cheat [hunk 4]
```diff
@@ -61,4 +61,19 @@ git rebase master -S -f
 # See all open pull requests of a user on Github
 url::open 'https://github.com/pulls?&q=author:<user>+is:open+is:pr'
 
+# Add a new module
+git submodule add <repository> [<path>]
```
Comment (denisidoro) [inline cheats/git.cheat]: hmm, this syntax won't work

**Comment by denisidoro** [comment]
I'll add some changes and merge this possibly on Monday, OK?

**Pull request merged**